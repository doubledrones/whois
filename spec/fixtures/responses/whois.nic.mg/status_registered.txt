Domain Name: google.mg
Domain ID: 1915-nicmg
WHOIS Server: NIC.MG
Referral URL: 
Updated Date: 2012-10-26T13:56:38.076Z
Creation Date: 2009-06-18T08:38:20.671Z
Registry Expiry Date: 2013-11-26T21:00:00.000Z
Sponsoring Registrar: MarkMonitor 
Sponsoring Registrar IANA ID: 
Domain Status: clientDeleteProhibited
Domain Status: clientTransferProhibited
Domain Status: clientRenewProhibited
Domain Status: ok
Domain Status: clientUpdateProhibited

Registrant ID: 4112-nicmg
Registrant Name: GOOGLE INC
Registrant Organization: GOOGLE INC
Registrant Street: Street Migrate
Registrant City: Antananarivo
Registrant State/Province: Antananarivo
Registrant Postal Code: 101
Registrant Country: MG

Tech ID: 4113-nicmg
Tech Name: Rafaralahisoa Emmanuel
Tech Organization: DTS
Tech Street: Immeuble Galaxy
Tech Street: Andraharo
Tech City: Antananarivo
Tech State/Province: 
Tech Postal Code: 101
Tech Country: MG
Tech Phone: +261 20 22 203 59
Tech Phone Ext: 
Tech Fax: +261 20 22 203 60
Tech Fax Ext: 

Name Server: ns1.google.com
Name Server: ns2.google.com
Name Server: ns3.google.com
Name Server: ns4.google.com

DNSSEC: unsigned


Additional Section


Sponsoring Registrar URL: http://www.markmonitor.com 
Sponsoring Registrar Address: MarkMonitor  Emerald Tech Center  391 N. Ancestor Place  Boise, ID 83704  
Sponsoring Registrar Country: US
Sponsoring Registrar Phone: +1 (208) 389-5740 
Sponsoring Registrar Fax: +1 (208) 389-5771 
Sponsoring Registrar Customer Service Contact: ccops@markmonitor.com 
Sponsoring Registrar Customer Service Email: ccops@markmonitor.com 
Sponsoring Registrar Admin Contact: Domain Billing 
Sponsoring Registrar Admin Email: ccops@markmonitor.com 


TERMS OF USE: You are not authorized to access or query our Whois
database through the use of electronic processes that are high-volume and
automated.  Whois database is provided by NIC-MG as a service to the internet
community on behalf of NIC-MG members. 

The data is for information purposes only. NIC-MG does not
guarantee its accuracy. By submitting a Whois query, you agree to abide
by the following terms of use: You agree that you may use this Data only
for lawful purposes and that under no circumstances will you use this Data
to: (1) allow, enable, or otherwise support the transmission of mass
unsolicited, commercial advertising or solicitations via e-mail, telephone,
or facsimile; or (2) enable high volume, automated, electronic processes
that apply to NIC-MG it's members (or NIC-MG or member computer systems). The
compilation, repackaging, dissemination or other use of this Data is
expressly prohibited.


>>> Last update of WHOIS database: 2013-07-01T09:19:01.999Z <<<
