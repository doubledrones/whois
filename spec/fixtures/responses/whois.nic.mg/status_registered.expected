#domain
  should: %s == "google.mg"

#domain_id
  should: %s == "1915-nicmg"


#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2009-06-18 08:38:20 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-10-26 13:56:38 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-11-26 21:00:00 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "MarkMonitor"
  should: %s.organization == nil
  should: %s.url          == "http://www.markmonitor.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.google.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.google.com"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.google.com"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.google.com"
